scipy/stats/_variation.py

```diff
@@ -1,94 +1,8 @@
 import numpy as np
-from scipy._lib._util import (_nan_allsame, _contains_nan,
-                              normalize_axis_index, _get_nan)
-from ._stats_py import _chk_asarray
+from scipy._lib._util import _nan_allsame, _get_nan
 from ._axis_nan_policy import _axis_nan_policy_factory
 
 
-def _nanvariation(a, *, axis=0, ddof=0, keepdims=False):
```

Comment:
Looks like a lot of effort went into this, and perhaps it can be resurrected someday if/when we want to allow `override={'nan_omit': False}` in the decorator. But for now, looks like this is no longer used.